sdk/security/keyvault/azkeys/internal/transforms.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package main
+
+import (
+	"log"
+	"os"
+	"regexp"
+)
+
+// removing client prefix from types
+func regexReplace(fileName string, regex string, replace string) {
+	file, err := os.ReadFile(fileName)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	r := regexp.MustCompile(regex)
+	file = r.ReplaceAll(file, []byte(replace))
+
+	err = os.WriteFile(fileName, file, 0644)
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+func main() {
+	// delete the version path param check (version == "" is legal for Key Vault but indescribable by OpenAPI)
```

Comment:
we've left OpenAPI behind, need something like
```suggestion
	// delete the version path param check (TypeSpec doesn't allow optional path parameters)
```

Reply:
fixed in 6a21e82